sparkle/src/stories/Page.stories.tsx

```diff
@@ -186,7 +186,7 @@ export const HelpExample = () => {
         icon={ChatBubbleBottomCenterTextIcon}
       />
       <Page.Vertical gap="xs" align="left">
-        <Page.SectionHeader title="Getting started?" visual={FlagStrokeIcon} />
+        <Page.SectionHeader title="Getting started?" />
```

Comment:
This is a bit weird to have a diff on sparkle and front in the same PR

Ideally they would be separated since in practice one won't be able to use the other

Reply:
just discussed with Ed and this property `visual` was not used anywhere, so no problem to take the shortcut of updating it without publishing